<launch>
    <env name="ROSCONSOLE_FORMAT" value="[${severity}] [${time}] [${node}]: ${message}"/>
    <param name="omnibot_platform_config" value="$(find omnibot_platform_drv)/config/omnibot_platform_config.yaml"/>
    <node pkg="omnibot_platform_drv" type="omnibot_platform_drv_node" name="omnibot_platform_drv_node" output="screen"/>
</launch>
